gbpservice/neutron/tests/unit/nfp/configurator/agents/test_firewall.py

```diff
@@ -144,7 +147,9 @@ def _test_handle_event(self, rule_list_info=True):
         """
 
         agent = self._get_FwHandler_objects()
-        driver = fw_dvr.FwaasDriver()
+        with mock.patch.object(cfg, 'CONF') as mock_cfg:
+            mock_cfg.configure_mock(rest_timeout='30', host='foo')
```

Comment:
Jagadish, as this is just a mocked variable, we can use any data. This is
similar to other variables like fip and mac address.
On 27-Mar-2016 22:02, "njagadish" <notifications@github.com> wrote:

> In gbpservice/neutron/tests/unit/nfp/configurator/agents/test_firewall.py
> <https://github.com/oneconvergence/group-based-policy/pull/147#discussion_r57531186>
> :
>
> > @@ -144,7 +147,9 @@ def _test_handle_event(self, rule_list_info=True):
> >          """
> >
> >          agent = self._get_FwHandler_objects()
> > -        driver = fw_dvr.FwaasDriver()
> > +        with mock.patch.object(cfg, 'CONF') as mock_cfg:
> > +            mock_cfg.configure_mock(rest_timeout='30', host='foo')
>
> Default rest_timeout is 360 sec right? why is this initialized to 30 sec
> here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/147/files/ccde939466fd9e4794fc768bdfd2a85b187cb0eb#r57531186>
>